canonical-websites/snapcraft.io
# Add sign in and account dropdown to navigation

Fixes #393

Adds sign in link and account dropdown to navigation.

### QA

- ./run or http://snapcraft.io-pr-498.run.demo.haus/
- /logout (just in case you are logged in from before)
- go to any page, 'Sign in' should be in header
- click 'Sign in'
- go through login screens
- you should be redirected back to snapcraft
- your user name should be in right side of header
- click you user name, dropdown will appear with links to account page and sign out
- go to 'Account details' ( account page should be opened)
- click 'Sign out' in dropdown, you should be signed out

Also check if it works on smaller screens.
<img width="1200" alt="screen shot 2018-04-18 at 10 14 27" src="https://user-images.githubusercontent.com/83575/38919614-4f0c9bc2-42f1-11e8-9026-9b7608579440.png">
<img width="1220" alt="screen shot 2018-04-12 at 15 23 51" src="https://user-images.githubusercontent.com/83575/38680280-35bafe78-3e66-11e8-8df1-53220ee65307.png">
<img width="1220" alt="screen shot 2018-04-12 at 15 23 58" src="https://user-images.githubusercontent.com/83575/38680279-359df936-3e66-11e8-82fe-ed828d5d4e01.png">
<img width="362" alt="screen shot 2018-04-11 at 14 11 29" src="https://user-images.githubusercontent.com/83575/38615987-d13f6bce-3d92-11e8-899e-533b8e5ec6a4.png">

With long user name:
<img width="1075" alt="screen shot 2018-04-17 at 12 52 42" src="https://user-images.githubusercontent.com/83575/38865516-48a97470-423e-11e8-979c-be6e4212d4ae.png">
